src/mame/machine/nb1412m2.cpp

```diff
@@ -294,7 +294,7 @@ void nb1412m2_device::timer_w(uint8_t data)
 	// When changes the DAC clock, Sound driver set wait loop count ($C010)
 	// in the range of 2 to 4 in order to keep the tempo of BGM even if changed clock.
 	// This value verified with BGM tempo of PCB.
-	m_timer->adjust(attotime::from_hz(446 * m_timer_rate));
+	m_timer->adjust(attotime::from_hz(450 * m_timer_rate));
```

Comment:
I’m still not too sure about this – you need 2*3*3*5*5 to get 450, while even 448 is simpler with 2*2*2*2*2*2*7.  Remember each factor of 2 can be achieved with a simple ripple counter stage while other factors require some kind of more complex synchronous logic.

Reply:
I see, that's helpful.
I set this one to 448.